extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -647,7 +644,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       return 0;
     });
     const renderableContacts = sortedContacts.slice(0, this.MAX_CONTACTS_LENGTH);
-    if ((renderableContacts.length > 0 || this.contactSearchInProgress) || !this.googleContactsSearchEnabled) {
+    if (renderableContacts.length > 0 || !this.googleContactsSearchEnabled) {
```

Comment:
I meant to remove the `if`, make this code run each time. We do need to show `No Contacts Found` at the end of the search, don't we?

Reply:
Oh we don't. Ok, then.